src/api_cod/subs/missing_exists.php

```
@@ -102,3 +102,215 @@ function exists_by_qids_query($endpoint_params)
+        ORDER BY 4 DESC;
+    SQL;
+    // ---
+    $params = [$category, $category];
+    // ---
+    return [$qua, $params];
+    // ---
+}
+
+
+
+function exists_statics_by_category($endpoint_params)
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$endpoint_params'. (undefined)

(UnusedFormalParameter)